Yosemite/YosemiteTests/Mocks/Networking/Remote/MockShippingLabelRemote.swift

```diff
@@ -6,6 +6,10 @@ import XCTest
 /// Mock for `ShippingLabelRemote`.
 ///
 final class MockShippingLabelRemote {
+
+    private(set) var purchaseShippingLabelCalled = false
+    private(set) var checkLabelStatusCallsCount = 0
```

Comment:
I like to set this kind of probing `var`s that allow tests to inspect the state of a test double as `private(set)` to prevent consumers from changing them. They should change only as a result of the dependency being doubled affecting the object.